docs/src/common_links.inc

```diff
@@ -69,5 +69,6 @@
 .. _@stephenworsley: https://github.com/stephenworsley
 .. _@tkknight: https://github.com/tkknight
 .. _@trexfeathers: https://github.com/trexfeathers
+.. _@TTV-Intrepid: https://github.com/TTV-Intrepid
```

Comment:
@ESadek-MO We tend to keep the `common_links.inc` for core devs, so could you remove this entry and add to the `whatsnew` (line 96) instead? Thanks.